sky/packages/sky/lib/src/widgets/gesture_detector.dart

```diff
@@ -27,7 +28,10 @@ class GestureDetector extends StatefulComponent {
     this.onHorizontalDragEnd,
     this.onPanStart,
     this.onPanUpdate,
-    this.onPanEnd
+    this.onPanEnd,
+    this.onScaleStart,
+    this.onScaleUpdate,
+    this.onScaleEnd
   }) : super(key: key);
```

Comment:
I have am adding a bunch of test code, going to add it in a separate CL so I can land this.